internal/privi/server.go

```diff
@@ -27,37 +27,27 @@ type PutRecordRequest struct {
 }
 
 type Server struct {
-	// TODO: allow privy server to serve many stores, not just one user
 	stores map[syntax.DID]*store
+	perms  permissions.Store
 	// Used for resolving handles -> did, did -> PDS
 	dir identity.Directory
 }
 
 // NewServer returns a privi server.
-func NewServer(didToStores map[syntax.DID]permissions.Store) *Server {
+func NewServer(
+	dids []syntax.DID,
+	permissionStore permissions.Store,
+) *Server {
 	server := &Server{
 		stores: make(map[syntax.DID]*store),
 		dir:    identity.DefaultDirectory(),
 	}
-	for did, perms := range didToStores {
-		err := server.Register(did, perms)
-		if err != nil {
-			log.Err(err)
-		}
+	for _, did := range dids {
+		server.stores[did] = newStore(did, permissionStore)
```

Comment:
hm ... this feels a bit weird but fine for now ... wonder if we want to be able to separate permission server from repo or if it should always go through